java-manta-client-kryo-serialization/src/main/java/com/joyent/manta/serialization/BaseBlockCipherSerializer.java

```diff
@@ -25,13 +25,14 @@
 import org.bouncycastle.jcajce.provider.symmetric.util.BlockCipherProvider;
 import org.bouncycastle.jcajce.spec.GOST28147ParameterSpec;
 import org.bouncycastle.jcajce.util.BCJcaJceHelper;
-import org.objenesis.instantiator.sun.MagicInstantiator;
+import org.objenesis.instantiator.sun.UnsafeFactoryInstantiator;
```

Comment:
The implementation of `MagicInstantiator` was using a security problem in the JDK to perform construction less instantiation and in Java 9 the "security problem" was resolved with no backwards compatibility added. I switched to using the `UnsafeFactoryInstantiator` because it uses the Java `unsafe` package which is still available in Java 9 and is also available across different JVM vendors.